en/components/grids_templates/column_resizing.md

```diff
@@ -167,7 +166,126 @@ column.autosize();
+
+``` scss
+$blue-color: #0288D1;
+$custom-grid-theme: igx-grid-theme(
+    $resize-line-color: $blue-color
+);
+
+```
+ >[!NOTE]
+ >If the component is using an [`Emulated`](../themes/component-themes.md#view-encapsulation) ViewEncapsulation, it is necessary to `penetrate` this encapsulation using `::ng-deep`.
+
+```scss
+:host {
+    ::ng-deep {
+        @include igx-grid($custom-grid-theme);
+    }
+}
+```
+
+#### Defining a color palette
+Instead of hard-coding the color values, we can achieve greater flexibility in terms of colors by using the [`igx-palette`]({environment:sassApiUrl}/index.html#function-igx-palette) and [`igx-color`]({environment:sassApiUrl}/index.html#function-igx-color) functions.
+
+`igx-palette` generates a color palette based on the specified primary and secondary color:
+
+```scss
+$primary-color: #0288D1;
+$secondary-color: #BDBDBD;
+
+$custom-theme-palette: igx-palette($primary: $primary-color, $secondary: $secondary-color);
+```
+
+And then, with [`igx-color`]({environment:sassApiUrl}/index.html#function-igx-color), we can easily retrieve the color from the palette. 
+
+```scss
+$custom-grid-theme: igx-grid-theme(
+    $palette: $custom-theme-palette,
+    $resize-line-color: igx-color($custom-theme-palette, 'secondary', 500)
+);
+```
+
+>[!NOTE]
+>The `igx-color` and `igx-palette` are powerful functions for generating and retrieving colors. Please, refer to [`Palettes`](../themes/palette.md) topic for detailed guidance on how to use them.
+
+#### Using Schemas
+Going further with the theming engine, you can build a robust and flexible structure that benefits from [**schemas**](../themes/schemas.md). A **schema** is a recipe of a theme.
+
+Extend the predefined schema provided for every component, in this case - [`light-grid`]({environment:sassApiUrl}/index.html#variable-_light-grid) schema:
+
+```scss
+// Extending the light grid schema
+$light-grid-schema: extend($_light-grid,
+    (
+        resize-line-color: (
+            igx-color: ('secondary', 500)
+            ),
+        header-background: (
+            igx-color: ("primary", 100)
+            ),
+        header-text-color: (
+            igx-color: ("primary", 600)
+            )
+    )
+);
+```
+
+In order to apply our custom schema, we have to **extend** one of the globals ([`light`]({environment:sassApiUrl}/index.html#variable-light-schema) or [`dark`]({environment:sassApiUrl}/index.html#variable-dark-schema)), which is basically pointing out the components with a custom schema, and after that add it to the respective component theme:
+
+```scss
+// Extending the global light-schema
+$custom-light-grid-schema: extend($light-schema,(
+    igx-grid: $light-grid-schema
+));
+
+// Specifying the palette and schema of the custom grid theme
+$custom-grid-theme: igx-grid-theme(
+    $palette: $custom-theme-palette,
+    $schema: $custom-light-grid-schema
+);
+```
+Don't forget to include the theme in the same way as it was demonstrated above.
+
```

Comment:
It will be better if there is a `####Demo` section, so that the reader could have faster reference to the sample.